src/App.tsx

```diff
@@ -12,28 +19,45 @@ import {
 } from "./contexts";
 import Router from "./Router";
 
+declare module "@mui/styles/defaultTheme" {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  interface DefaultTheme extends Theme {}
+}
+
+// https://mui.com/material-ui/customization/color/#2014-material-design-color-palettes
+const theme = createTheme({
+  palette: {
+    primary: indigo,
+    secondary: purple,
+  },
+});
+
 function App() {
   return (
-    <SnackbarProvider maxSnack={3}>
-      <UserProvider>
-        <ProjectProvider>
-          <BuildProvider>
-            <TestRunProvider>
-              <SocketProvider>
-                <HelpProvider>
-                  <Box height="10%">
-                    <Header />
-                  </Box>
-                  <Box height="90%">
-                    <Router />
-                  </Box>
-                </HelpProvider>
-              </SocketProvider>
-            </TestRunProvider>
-          </BuildProvider>
-        </ProjectProvider>
-      </UserProvider>
-    </SnackbarProvider>
+    <StyledEngineProvider injectFirst>
+      <ThemeProvider theme={theme}>
+        <SnackbarProvider maxSnack={3}>
+          <UserProvider>
+            <ProjectProvider>
+              <BuildProvider>
+                <TestRunProvider>
+                  <SocketProvider>
+                    <HelpProvider>
+                      <Box height="4.5rem">
```

Comment:
`Box` element seems now consume properties differently 
try this
```
<Box sx={{ height: "10%" }}>
```
https://mui.com/material-ui/react-box/